main.go

```diff
@@ -57,15 +58,22 @@ func main() {
 						Name:  "playlist",
 						Usage: "Playlist file (empty for LCDZ playlist)",
 					},
+					&cli.StringFlag{

```

Comment:
I switched these into uint flags, and popped a constant at the top of the file that is used as default values - the CLI package is pretty good at self documenting, so I've let that do its job :)